catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -14,13 +14,21 @@
  -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
     <!-- The tika input transformer programmatically registers itself as a service -->
-    <bean id="tikaTransformer" class="ddf.catalog.transformer.input.tika.TikaInputTransformer">
+    <bean id="tikaTransformer" class="ddf.catalog.transformer.input.tika.TikaInputTransformer" init-method="populateMimeTypeMap">
         <argument ref="blueprintBundleContext"/>
         <argument ref="commonMetacardType"/>
+        <property name="someMetacardType" ref="someMetacardType"/>
+        <property name="jpegMetacardType" ref="jpegMetacardType"/>
+        <property name="excelMetacardType" ref="excelMetacardType"/>
+        <property name="mp4MetacardType" ref="mp4MetacardType"/>
+        <property name="mpegMetacardType" ref="mpegMetacardType"/>
+        <property name="officeDocMetacardType" ref="officeDocMetacardType"/>
+        <property name="pdfMetacardType" ref="pdfMetacardType"/>
+        <property name="powerpointMetacardType" ref="powerpointMetacardType"/>
     </bean>
 
     <reference id="commonMetacardType" interface="ddf.catalog.data.MetacardType"
-               filter="(name=common)"
+               filter="(name=fallback.common)"
```

Comment:
See other comment about the common metacard type. I think we should just create our own fallback type here that mirrors the common type so we don't rename the non-tika common metacard type.